README.md

```diff
@@ -397,6 +397,7 @@ There're also some great commercial libraries, like [amchart](https://www.amchar
 * [mout](https://github.com/mout/mout) - Modular JavaScript Utilities.
 * [mesh](https://github.com/crcn/mesh.js) - Streamable data synchronization utility.
 * [preludejs](https://github.com/alanrsoares/prelude-js) - Hardcore Functional Programming for JavaScript.
+* [rambda](https://github.com/selfrefactor/rambda) - Faster and smaller alternative to *Ramda*.
```

Comment:
I always wonder why people create their versions instead of contributing to the original one. unless the author rejects all contributions, which's completely against open source in the first place, but still.

Reply:
Thank you for accepting this PR. In this case, the original reason was that Ramda was very bad at tree-shaking and the solution was not straightforward and that is how Rambda was created. Otherwise, I agree that it is not something to admire(create own version instead of contributing to to original one).